.github/workflows/nightly-builds.yml

```
@@ -182,7 +182,7 @@ jobs:
 
       - name: Publish
         # Publish (osgi bundle) not working with JDK 17, issue #31132
-        if: ${{ startsWith(matrix.jdkVersion, 'adopt@1.8') || ${{ startsWith(matrix.jdkVersion, 'adopt@1.11') }}
+        if: ${{ startsWith(matrix.jdkVersion, 'adopt@1.8') || startsWith(matrix.jdkVersion, 'adopt@1.11') }}
```

Oops, easy to miss.

I guess we should perhaps test this with jdk11 only, since that's what we're using for the actual publishing anyway.

```suggestion
        if: ${{ startsWith(matrix.jdkVersion, 'adopt@1.11') }}
```

---

agreed